src/Avalonia.Controls/TextBox.cs

```diff
@@ -535,18 +535,18 @@ public string RemoveInvalidCharacters(string text)
             return text;
         }
 
-        public async void Cut()
+        public async System.Threading.Tasks.Task Cut()
         {
             var text = GetSelection();
             if (text is null) return;
 
             _undoRedoHelper.Snapshot();
-            Copy();
+            await Copy();
             DeleteSelection();
             _undoRedoHelper.Snapshot();
         }
 
-        public async void Copy()
+        public async System.Threading.Tasks.Task Copy()
```

Comment:
I don't think we should be making changes like this in a "fix warnings PR"

Reply:
I regard warnings as errors.
In this case they can lead to unwanted behavior.

```
TextBox.cs(538,27): warning CS1998: This async method lacks 'await' operators and will run synchronously. Consider using the 'await' operator to await non-blocking API calls, or 'await Task.Run(...)' to do CPU-bound work on a background thread. [C:\GitHub\Avalonia\src\Avalonia.Controls\Avalonia.Controls.csproj]
```

If you want to reverse the commit.